src/core/sys/posix/sys/types.d

```diff
@@ -731,9 +726,22 @@ else version (Solaris)
 
     alias uint pthread_key_t;
 }
-else version( Android )
+else version( CRuntime_Bionic )
 {
-    version(X86)
+    version(D_LP64)
+    {
+        struct pthread_attr_t
+        {
+            uint    flags;
+            void*   stack_base;
+            size_t  stack_size;
+            size_t  guard_size;
+            int     sched_policy;
+            int     sched_priority;
+            char[16] __reserved;
+        }
+    }
+    else
```

Comment:
Make this `else version(X86)` or something, because you have two else clauses now, which is causing the auto-tester to fail.  Also, you'll need to rebase against the latest version of #1010.